### PR TITLE
Convert project profile card update template into github issue form

### LINKED ISSUE #2292: Convert Project Profile Card review and update Template to Github Form Issue
### Overview
As an issue creator, we want to implement issue templates for issue creation to streamline and simplify the process. The current issue forms require the person to replace the text, and is less user-friendly compared to GitHub Form issues.

### Action Items
- [x] Define the form elements and their behavior
  - [x] Review with product and Front End Lead
- [x] Create a project-profile-card-review-and-update.yml file in your own fork in: ```gh-pages/.github/ISSUE_TEMPLATE```
- [x] Follow github form syntax readings below to code a similar form to the [current issue template](https://github.com/hackforla/website/issues/new?assignees=&labels=&template=project-profile-card-review-and-update.md&title=Update+Project+Profile%3A+%5BProject+Name%5D).
- [x] Get Signoff by Product on the proposed template before making a PR
- [x] Delete project-profile-card-review-and-update.md file in [this folder](https://github.com/hackforla/website/tree/gh-pages/.github/ISSUE_TEMPLATE)

### Resources/Instructions
[Current Issue Template Code File](https://github.com/hackforla/website/blob/gh-pages/.github/ISSUE_TEMPLATE/project-profile-card-review-and-update.md)
[Github Forms Issue Reading](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms)
[Github Forms Issue Reading Pt 2](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema)
#2128 

To get signoff from product: put your issue in the Review column on the project board.

### PR DESCRIPTION
**Pull request by MattPereira**
Fixes #2292

### What changes did you make and why did you make them ?

  - Created a GitHub issue form to replace the current markdown issue template in order to streamline and simplify the process of issue creation. The user creating the issue will no longer have to replace text in markdown format.
  - Converted "Project Name" and "Project Card URL" into drop-downs with one option for every HFLA project
  - Converted action item checklist from [current issue template](https://github.com/hackforla/website/blob/gh-pages/.github/ISSUE_TEMPLATE/project-profile-card-review-and-update.md) into yes/no drop-downs followed by an input area (as requested at the Sunday meeting on 10/23/22).
  - Converted "status" item to a dropdown with the options: "active", "completed", or "on hold".
  - Deleted the old markdown template
  - Included "Additional Instructions" prompting issue creator to add the appropriate label and add the issue to the project board (as requested at Sunday meeting on 11/13/22).

<details>
  <summary><strong>Click here</strong> to see the additional instructions</summary>

</details>

### CODE REVIEW

**Comment by DmitriiTsy** [comment]
ETA: Morning 10/29/22
Availability: 4 hours

**Review by DmitriiTsy — DISMISSED** [review]
@MattPereira 
- All the changes look correct 
- Linters are okay as well as all checks 
You did a really great job. 👍

**Comment by esantiano** [comment]
ETA: 10/31/22
Availability: 3-5PM

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+        - "Home Unite Us"
+        - "LA TDM Calculator"
+        - "Lucky Parking"
+        - "Open Community Survey"
+        - "VRMS"
+        - "Write for All"
+        - "Youth Justice Nav"
+        - "EMS Triage Tracker"
+        - "Heart"
+        - "HelloGov"
```
Comment (esantiano) [inline .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml]: HelloGov to HelloGOV

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+    id: project-card-url
+    attributes:
+      label: Project Card URL
+      description: "https://github.com/hackforla/website/tree/gh-pages/_projects"
+      options: 
+        - "100-automations.md"
+        - "311-data.md"
+        - "access-the-data.md"
+        - "adopt-civic-art.md"
+        - "brigade-organizers-playbook.md"
```
Comment (esantiano) [inline .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml]: switch lines with ballot-nav.md for alphabetical order

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+        - "Heart"
+        - "HelloGov"
+        - "Public Tree Map"
+        - "Railstats LA"
+        - "Undebate"
+        - "Work for LA"
+        - "Jobs for Hope"
+        - "New Schools Today"
+        - "Not Today - Self-Defense Against Suicidal Thoughts"
+        - "Spare"
```
Comment (esantiano) [inline .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml:51]: Include an "other" option for unlisted projects that have an existing project url

**Review by esantiano — CHANGES_REQUESTED** [review]
Hey @MattPereira, 

The new issue template looks way better compared to the previous, I just had a few suggestions in regards to the spelling, ordering, and options available in the Project Name field.

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml (hunk 3 above)
Comment (gstemmann) [reply .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml:51]: Your notes for @MattPereira for changes, I think, are great. 
For adding "other" to the projects list, I wonder if that may lead to multiple issue creators mistakenly using "other" when the projects are already listed.

**Comment by gstemmann** [comment]
ETA: Morning 11/4/22
Availability: Tuesday 6-9pm

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml (hunk 3 above)
Comment (MattPereira) [reply .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml:51]: I wonder if the "project profile card update" issue template is only used for updating the "Current Projects" listed on the [hfla projects page](https://www.hackforla.org/projects/) or if this issue form is also used for other internal record keeping not displayed on the website.

**Comment by MattPereira** [comment]
> Hey @MattPereira,
> 
> The new issue template looks way better compared to the previous, I just had a few suggestions in regards to the spelling, ordering, and options available in the Project Name field.

Hi @esantiano ,

I appreciate the kind words and astute review. Great catch on the casing of HelloGOV and the alphabetical listing error. I am also open to discussing how to better handle the potential edge case of unlisted projects.

**Review by esantiano — DISMISSED** [review]
Hey @MattPereira, 

Thanks for addressing my comments, this looks good to me. Keep up the good work!

**Comment by luiderek** [comment]
From a UX perspective, having a drop-down to reveal a yes/no split seems frustrating. Is the checkbox possibly something than can be used? I know it's something that exists in issue forms, but not sure how nicely interaction cooperates with them. 
It's something that'd only save a click, but with 14 mandatory drop-downs I think they add up.

**Comment by MattPereira** [comment]
> From a UX perspective, having a drop-down to reveal a yes/no split seems frustrating. Is the checkbox possibly something than can be used? I know it's something that exists in issue forms, but not sure how nicely interaction cooperates with them. It's something that'd only save a click, but with 14 mandatory drop-downs I think they add up.

I agree with you 100% about having so many yes/no drop-downs being an unpleasant user experience. However, as noted in the pull request creation comment: 
>- Converted action item checklist from [current issue template](https://github.com/hackforla/website/blob/gh-pages/.github/ISSUE_TEMPLATE/project-profile-card-review-and-update.md) into yes/no drop-downs followed by an input area as requested by Bonnie at the Sunday meeting.

I used Bonnie's [communities-of-practice-information-updates.yml](https://github.com/hackforla/website/blob/gh-pages/.github/ISSUE_TEMPLATE/communities-of-practice-information-updates.yml) as a guide to build this new form.

**Review by gstemmann — DISMISSED** [review]
The new form looks great!  And much cleaner than the look with the checkboxes. Great Job!

**Comment by jyaymie** [comment]
Review ETA: 11/7/22 9am
Availability: 11/7/22 Now - 9am

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml [hunk 4]
```diff
@@ -0,0 +1,330 @@
+name: "Project Profile Card review and update"
+description: "To track project profile card updates"
+title: "Update Project Profile: [Project Name]"
+labels: ["P-Feature: Project Info and Page", "role: back end/devOps, role: front end, Size: Small, size: 0.5pt, time sensative"]
```
Comment (jyaymie) [inline .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml]: I'd remove the end quotation in `"P-Feature: Project Info and Page",` to improve readability and correct the typo from `sensative` to `sensitive`.

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+      label: Update the Technologies?
+      options: 
+        - "YES"
+        - "NO"
+    validations:
+      required: true
+  - type: textarea
+    id: technologies
+    attributes:
+      label: Techonologies
```
Comment (jyaymie) [inline .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml]: Looks like an extra "o" snuck in the Technologies label.

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml [hunk 6]
```diff
@@ -0,0 +1,330 @@
...
+      label: "Remove person(s) and info from Leadership?"
+      options:
+        - "YES"
+        - "NO"
+    validations:
+      required: true
+  - type: textarea
+    id: remove-leadership
+    attributes:
+      label: "If YES on remove person(s) edit the below to provide the information required. If NO, delete the template in the field below:"
```
Comment (jyaymie) [inline .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml]: Grammar-wise, I think there should be a comma after `person(s)`.

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml [hunk 7]
```diff
@@ -0,0 +1,330 @@
...
+        - "Completed"
+        - "On Hold"
+    validations:
+      required: true
+      
+  - type: dropdown
+    id: update-meeting-times-yes-no
+    attributes: 
+      label: Update your meeting times?
+      options:
```
Comment (jyaymie) [inline .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml:315]: Pickiest comment of all! For consistency's sake, I'd put these dropdown options in all caps.

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml [hunk 8]
```diff
@@ -0,0 +1,330 @@
...
+title: "Update Project Profile: [Project Name]"
+labels: ["P-Feature: Project Info and Page", "role: back end/devOps, role: front end, Size: Small, size: 0.5pt, time sensative"]
+
+body:
+  - type: markdown
+    attributes:
+      value: "# Overview"
+  - type: markdown
+    attributes:
+      value: "As a product manager, I want my project's details to be up to date on the HackforLA website."
```
Comment (jyaymie) [inline .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml]: I'm not sure if `HackforLA` needs to have spaces between the words? I know no spaces were included in the original card, but I don't see the organization's name written like that anywhere on the website.

**Review by jyaymie — CHANGES_REQUESTED** [review]
Such thorough work, @MattPereira! You followed all the items in the linked issue, including making a comment with your defined form elements and behavior, and I witnessed how highly communicative you were with the team throughout this entire process. I requested some very minor edits, but wow! Impressive job.

Edit: I'm curious! How did you manage to default the "Update Test Site Link?" dropdown to `Selection: None`. I noticed it in the link to your GH form, but I couldn't find the code for it.

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml (hunk 4 above)
Comment (MattPereira) [reply .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml]: Nice catch on the readability issue here

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml (hunk 5 above)
Comment (MattPereira) [reply .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml]: I definitely should have thought to turn on a spell checking extension for this issue :facepalm:

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml (hunk 6 above)
Comment (MattPereira) [reply .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml]: Yes there absolutely should be!

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml (hunk 7 above)
Comment (MattPereira) [reply .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml:315]: I couldn't agree more!

File: .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml (hunk 8 above)
Comment (MattPereira) [reply .github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml]: I'm on team spaces with you!

**Comment by MattPereira** [comment]
> Edit: I'm curious! How did you manage to default the "Update Test Site Link?" dropdown to `Selection: None`. I noticed it in the link to your GH form, but I couldn't find the code for it.

Oops, this was actually an error. It looks like GitHub issue forms defaults dropdowns to `Selection:None` if you do not include `required: true` for the input's validations. 

As always, I appreciate your generous compliments and superbly meticulous reviews @jyaymie!

**Comment by jyaymie** [comment]
> > Edit: I'm curious! How did you manage to default the "Update Test Site Link?" dropdown to `Selection: None`. I noticed it in the link to your GH form, but I couldn't find the code for it.
> 
> Oops, this was actually an error. It looks like GitHub issue forms defaults dropdowns to `Selection:None` if you do not include `required: true` for the input's validations.
> 
> As always, I appreciate your generous compliments and superbly meticulous reviews @jyaymie!

Learned something new. Way to figure that out!

**Review by jyaymie — DISMISSED** [review]

**Review by jdingeman — COMMENTED** [review]
Hi @MattPereira, this looks very good. Nice job. Did you bring this up with Product yet to confirm the design?

**Comment by MattPereira** [comment]
@jdingeman requested I tag @ExperimentsInHonesty to add this to the Sunday agenda

**Review by DmitriiTsy — DISMISSED** [review]

**Review by jdingeman — APPROVED** [review]
Nice job @MattPereira! Thanks for your patience. After commenting on #2292, putting the appropriate labels, and moving to New Issue Approval, you can move on to your next issue.

**Pull request merged**